src/components/CTASpeakers/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Img from 'gatsby-image';
+import { shape, object } from 'prop-types';
+import { Link } from 'gatsby';
+
+import CTASpeakersBgSVG from '../svg/CTASpeakersBgSVG';
+
+import './index.scss';
+
+const CTASpeakers = ({ img }) => (
+  <div className="cta-speakers">
+    <div className="container flex-helper">
```

Comment:
Bootstrap 4 already uses flex for columns. I would try using those, instead of `flex-helper`, as they are already styled for responsiveness.